travis-infrastructure/terraform-config
# Dramatically reduce precise container-based capacity

## What is the problem that this PR is trying to fix?

Since turning on the sudo-enabled rollout for `dist: precise` jobs, we have seen a dramatic decrease in demand for container-based precise, meaning that we are typically running with headroom of ~75, which is roughly equivalent to 6 c3.8xlarge instances sitting idle, which is 💸 .

## What approach did you choose and why?

Reduce min and max ASG sizes, reduce scaling step to 1, reduce scale in and scale out triggers.

## How can you test this?

If the terraform apply does not work, we can mutate the ASGs directly, and then observe automatic capacity adjustments afterward.
